### PR TITLE
#2865 Update master branch's SNAPSHOT version to 0.9.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2865)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/rest/rest-workers.rst [hunk 1]
```diff
@@ -717,7 +717,7 @@ Example Response
 
             },
             "offsetTopicName":"f1f6ae812c",
-            "imageName":"oharastream/connect-worker:0.8.0-SNAPSHOT",
+            "imageName":"oharastream/connect-worker:0.9.0-SNAPSHOT",
```
Comment (konekoya) [inline docs/src/rest/rest-workers.rst]: @wu87988622 I don't see this in #2260. Are these versions should be updated as well?

File: docs/src/rest/rest-workers.rst [hunk 2]
```diff
@@ -717,7 +717,7 @@ Example Response
```
Comment (konekoya) [inline docs/src/rest/rest-workers.rst]: Test

File: docs/src/rest/rest-workers.rst (hunk 1 above)
Comment (vitojeng) [reply docs/src/rest/rest-workers.rst]: No need. ​These information just some **examples** ^^

Comment (chia7712) [reply docs/src/rest/rest-workers.rst]: I'd say just remove the "image" from this document.

Comment (konekoya) [reply docs/src/rest/rest-workers.rst]: No problem. I'm removing them to avoid the confusion in the future

**Comment by chia7712** [comment]
This PR can't pass since there is no 0.9.0-SNAPSHOT tag now. Just merge it after QA fails.

**Comment by konekoya** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by konekoya** [comment]
Thanks for the help @chia7712 💯

**Comment by konekoya** [comment]
@jackyoh please help build fresh images of 0.9.0-SNAPSHOT :)

**Comment by jackyoh** [comment]
> @jackyoh please help build fresh images of 0.9.0-SNAPSHOT :)

OK, Thank you.

**Pull request merged**